R/update-input.R

```diff
@@ -460,14 +479,14 @@ updateSliderInput <- function(session = getDefaultReactiveDomain(), inputId, lab
   }
 
   message <- dropNulls(list(
-    label = label,
-    value = formatNoSci(value),
-    min = formatNoSci(min),
-    max = formatNoSci(max),
-    step = formatNoSci(step),
-    `data-type` = dataType,
-    `time-format` = timeFormat,
-    timezone = timezone
+    label=processDeps(label, session),
```

Comment:
With apologies for the drive-by nitpick, can we keep the spaces in these changes?

```suggestion
    label = processDeps(label, session),
```

Reply:
Sure, I was not sure what to do there's a bit of both in there and I can't see any `.lintr`

Reply:
Done!